proto/query.proto

```diff
@@ -731,6 +731,9 @@ message StreamHealthResponse {
 
   // realtime_stats contains information about the tablet status
   RealtimeStats realtime_stats = 4;
+
+  // the alias as understood by the sending tablet
```

Comment:
nit: We're using Go style comments for the .proto file as well. How about documenting the purpose of this field here?

Suggestion:

// tablet_alias is the alias of the sending tablet. The discovery/healthcheck.go code uses it to verify that it's talking to the correct tablet and that it hasn't changed in the meantime e.g. due to tablet restarts where ports have been reused but assigned differently.